roles/pulp_webserver/handlers/main.yml

```diff
@@ -6,9 +6,13 @@
     daemon_reload: true
   become: true
 
-- name: reload Apache
+- name: reload apache
```

Comment:
nitpick: s/apache/Apache/

Reply:
Actually this change happend on purpose. It is done so we can "automatically" call the reload handler based on the server name.

```
---                                                                                                                                                             
- name: Import specified TLS certificate
  copy:
    src: "{{ pulp_webserver_ssl_cert }}"
    dest: "{{ pulp_webserver_tls_folder }}/pulp_webserver.crt"
    owner: root
    group: pulp
    mode: 0600
  notify: reload {{ pulp_webserver_server }}

- name: Import specified TLS private key 
  copy:
    src: "{{ pulp_webserver_ssl_key }}"
    dest: "{{ pulp_webserver_tls_folder }}/pulp_webserver.key"
    owner: root
    group: pulp
    mode: 0600
  notify: reload {{ pulp_webserver_server }}
```

If you really want `Apache` instead of `apache` It is fine, I can update the above with `{{ pulp_webserver_server | lower }}` 0 but I wanted to provide context of why this change happened in the first place.

@bmbouter your call. I am fine eithe way, simply let me know.

Reply:
Thanks for the clarification.